.core_files.yaml

```diff
@@ -0,0 +1,120 @@
+# Defines a list of files that are part of main core of Home Assistant.
+# Changes to these files/filters define how our CI test suite is ran.
+core: &core
+  - homeassistant/*.py
+  - homeassistant/auth/**
+  - homeassistant/helpers/*
+  - homeassistant/package_constraints.txt
+  - homeassistant/util/*
+  - pyproject.yaml
+  - requirements.txt
+  - setup.cfg
+
+# Our base platforms, that are used by other integrations
+base_platforms: &base_platforms
+  - homeassistant/components/air_quality/*
+  - homeassistant/components/alarm_control_panel/*
+  - homeassistant/components/binary_sensor/*
+  - homeassistant/components/button/*
+  - homeassistant/components/camera/*
+  - homeassistant/components/climate/*
+  - homeassistant/components/cover/*
+  - homeassistant/components/device_tracker/*
+  - homeassistant/components/fan/*
+  - homeassistant/components/humidifier/*
+  - homeassistant/components/image_processing/*
+  - homeassistant/components/light/*
+  - homeassistant/components/lock/*
+  - homeassistant/components/media_player/*
+  - homeassistant/components/notify/*
+  - homeassistant/components/number/*
+  - homeassistant/components/remote/*
+  - homeassistant/components/scene/*
+  - homeassistant/components/select/*
+  - homeassistant/components/sensor/*
+  - homeassistant/components/siren/*
+  - homeassistant/components/stt/*
+  - homeassistant/components/switch/*
+  - homeassistant/components/tts/*
+  - homeassistant/components/vacuum/*
+  - homeassistant/components/water_heater/*
+  - homeassistant/components/weather/*
+
+# Extra components that trigger the full suite
+components: &components
+  - homeassistant/components/alert/*
+  - homeassistant/components/alexa/*
+  - homeassistant/components/auth/*
+  - homeassistant/components/automation/*
+  - homeassistant/components/cloud/*
+  - homeassistant/components/config/*
+  - homeassistant/components/configurator/*
+  - homeassistant/components/conversation/*
+  - homeassistant/components/demo/*
+  - homeassistant/components/device_automation/*
+  - homeassistant/components/dhcp/*
+  - homeassistant/components/dhcp/*
+  - homeassistant/components/discovery/*
+  - homeassistant/components/energy/*
+  - homeassistant/components/ffmpeg/*
+  - homeassistant/components/frontend/*
+  - homeassistant/components/google_assistant/*
+  - homeassistant/components/group/*
+  - homeassistant/components/hassio/*
+  - homeassistant/components/homeassistant/**
+  - homeassistant/components/homekit/*
```

Comment:
Nothing depends on homekit itself so I'm not sure we need this
```suggestion
```

Reply:
It does, discovery

Reply:
Isn't homekit discovery fully contained in `zeroconf`?
https://github.com/home-assistant/core/blob/dev/homeassistant/components/zeroconf/__init__.py#L442

Reply:
Oh true! 👍